bin/ulauncher-toggle

```diff
@@ -1,13 +1,2 @@
 #!/bin/sh
-
-dbus-send \
-    --session \
-    --print-reply \
-    --dest=net.launchpad.ulauncher \
-    /net/launchpad/ulauncher \
-    net.launchpad.ulauncher.toggle_window
-
-# dbus-send is asynchronous and sometimes it takes a bit of time for Ulauncher window to be presented
-sleep 0.03
-
-wmctrl -a "Ulauncher - Application Launcher"
+ulauncher
```

Comment:
You might `exec` here on the off chance ulauncher isn't actually running and this starts it.

Reply:
Ok